desktop/core/src/desktop/settings.py

```diff
@@ -412,7 +412,11 @@
         'BACKEND': 'django.core.cache.backends.locmem.LocMemCache', # TODO: Parameterize here for all the caches
         'LOCATION': 'unique-hue'
     },
+    'axes_cache': {
+        'BACKEND': 'django.core.cache.backends.dummy.DummyCache',
+    },
 }
+AXES_CACHE = 'axes_cache'
```

Comment:
Because of this error `django.core.exceptions.ImproperlyConfigured: django-axes does not work properly with LocMemCache as the default cache backend please add e.g. a DummyCache backend for axes and configure it with AXES_CACHE
`

Reply:
after reading this https://django-axes.readthedocs.io/en/latest/4_configuration.html#configuring-caches

Reply:
:+1: We can start with Dummy, then Redis later if needed

https://docs.djangoproject.com/en/3.1/topics/cache/